app/services/mongodb/MongoDBQueueService.scala

```diff
@@ -68,7 +69,18 @@ trait MongoDBQueueService {
 
   // get next entry from queue
   def getNextQueuedAction(): Option[QueuedAction] = {
-    return Queue.findOne(new MongoDBObject)
+    try {
+      val response = Queue.findOne(new MongoDBObject)
+      response
+    } catch {
+      case e: MongoException => {
+        // TODO: Will generate an error message every 5ms while Mongo is inaccessible...
+        //Logger.error("Problem connecting to MongoDB queue.")
```

Comment:
Could we set the level to trace? For those cases when we do want to see the errors?

Reply:
kept as error, but with a new flag so it only prints once per broken connection.